bitnami/mongodb/templates/standalone/dep-sts.yaml

```diff
@@ -366,11 +366,21 @@ spec:
               /bin/mongodb_exporter --web.listen-address ":{{ .Values.metrics.containerPort }}" --mongodb.uri "{{ include "mongodb.mongodb_exporter.uri" . }}" {{ .Values.metrics.extraFlags }}
           env:
           {{- if .Values.auth.enabled }}
+          {{- if not (or .Values.metrics.username .Values.metrics.password ) }}
```

Comment:
Same here, if someone leaves metrics.password with a value but has metrics.username empty, that could cause some issue. We could just check for metrics.username instead.

Reply:
I removed `.Values.metrics.password` from the query